### PR TITLE
People Update GA

### PR DESCRIPTION
**Pull request by bfossen-ce**
People Update GA

### CODE REVIEW

File: data/ga/retired/Alan-Powell-bb48e87c-29db-4a4d-a278-fa299be2020e.yml [hunk 1]
```diff
@@ -8,18 +8,10 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/powell-alan-
 party:
```
Comment (maxknutsen) [inline data/ga/retired/Alan-Powell-bb48e87c-29db-4a4d-a278-fa299be2020e.yml:8]: @sroomf needs to be moved up to the legislature folder.

File: data/ga/retired/Becky-Evans-f9d51075-c77f-4c05-88b9-ed716fd3b964.yml [hunk 2]
```diff
@@ -8,16 +8,10 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/evans-becky-
 party:
```
Comment (maxknutsen) [inline data/ga/retired/Becky-Evans-f9d51075-c77f-4c05-88b9-ed716fd3b964.yml:8]: @sroomf move to leg folder.

File: data/ga/retired/Carolyn-Hugley-0f5a9273-2fc6-458f-af59-b75070a1dc76.yml [hunk 3]
```diff
@@ -8,16 +8,10 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/hugley-carol
 party:
```
Comment (maxknutsen) [inline data/ga/retired/Carolyn-Hugley-0f5a9273-2fc6-458f-af59-b75070a1dc76.yml:8]: @sroomf move to leg.

File: data/ga/retired/Chris-Erwin-d79c3e9b-b73d-416b-9bfd-8a0338d8256c.yml [hunk 4]
```diff
@@ -8,16 +8,10 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/erwin-chris-
 party:
```
Comment (maxknutsen) [inline data/ga/retired/Chris-Erwin-d79c3e9b-b73d-416b-9bfd-8a0338d8256c.yml:8]: @sroomf move to leg.

File: data/ga/legislature/Colton-Moore-33b353c3-50e9-485f-b154-0d5310da4f2f.yml [hunk 5]
```diff
@@ -0,0 +1,28 @@
+id: ocd-person/33b353c3-50e9-485f-b154-0d5310da4f2f
+name: Colton Moore
+given_name: Colton
+family_name: Moore
+email: colton.moore@senate.ga.gov
+party:
+- name: Republican
+roles:
```
Comment (maxknutsen) [inline data/ga/legislature/Colton-Moore-33b353c3-50e9-485f-b154-0d5310da4f2f.yml:8]: @sroomf this person served in the Georgia House District 1 from Jan 2018 until Jan 2020. So I think they should already have a file but it wouldn't be in this pull? @bfossen-ce solved some of these when we were in Chicago.

File: data/ga/legislature/Danny-Mathis-0db72b1b-2be3-4596-81b2-0bdd733c9bce.yml [hunk 6]
```diff
@@ -0,0 +1,38 @@
+id: ocd-person/0db72b1b-2be3-4596-81b2-0bdd733c9bce
+name: Danny Mathis
+given_name: Danny
+family_name: Mathis
+email: danny.mathis@house.ga.gov
+image: https://www.legis.ga.gov/api/images/default-source/portraits/mathis-danny-4959.jpg
```
Comment (maxknutsen) [inline data/ga/legislature/Danny-Mathis-0db72b1b-2be3-4596-81b2-0bdd733c9bce.yml:6]: @sroomf this person served in GA House District 144 from January 2019 until 01/09/2023 when they began serving in district 149. I don't see their original file in this pull but I do see they show up on openstates. https://openstates.org/search/?query=danny+mathis&state=ga

File: data/ga/legislature/Deborah-Silcox-b423e42d-eead-4f58-b82a-7c76a1a311b4.yml [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+name: Deborah Silcox
+given_name: Deborah
+family_name: Silcox
+email: deborah.silcox@house.ga.gov
+image: https://www.legis.ga.gov/api/images/default-source/portraits/silcox-deborah-4887.jpg
+party:
+- name: Republican
+roles:
+- type: lower
+  jurisdiction: ocd-jurisdiction/country:us/state:ga/government
```
Comment (maxknutsen) [inline data/ga/legislature/Deborah-Silcox-b423e42d-eead-4f58-b82a-7c76a1a311b4.yml:11]: @sroomf this person served in GA House district 52 from Jan 2019 until Jan 2021.

File: data/ga/retired/Derek-Mallow-73918c45-83c1-4d52-a84d-3ad7ad8de40b.yml [hunk 8]
```diff
@@ -8,13 +8,10 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/mallow-derek
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2023-01-11'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:ga/government
   district: '163'
```
Comment (maxknutsen) [inline data/ga/retired/Derek-Mallow-73918c45-83c1-4d52-a84d-3ad7ad8de40b.yml:14]: @sroomf move to leg.

File: data/ga/retired/Doreen-Carter-818f1afb-2850-448e-99c1-be63a9acfe95.yml [hunk 9]
```diff
@@ -8,16 +8,10 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/carter-doree
 party:
```
Comment (maxknutsen) [inline data/ga/retired/Doreen-Carter-818f1afb-2850-448e-99c1-be63a9acfe95.yml:8]: @sroomf move to leg.

File: data/ga/retired/Ed-Setzler-777f58bb-9d6b-4755-ab7b-b94d11629cb5.yml [hunk 10]
```diff
@@ -8,16 +8,10 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/setzler-ed-2
 party:
 - name: Republican
```
Comment (maxknutsen) [inline data/ga/retired/Ed-Setzler-777f58bb-9d6b-4755-ab7b-b94d11629cb5.yml:9]: @sroomf move to leg.

File: data/ga/retired/Emory-Dunahoo-3b2b9f31-6859-4582-8d82-581597c0c178.yml [hunk 11]
```diff
@@ -8,16 +8,10 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/dunahoo-emor
 party:
```
Comment (maxknutsen) [inline data/ga/retired/Emory-Dunahoo-3b2b9f31-6859-4582-8d82-581597c0c178.yml:8]: @sroomf move to leg.

File: data/ga/retired/Houston-Gaines-e73cb2a6-bf3e-4211-a386-142c18a4f607.yml [hunk 12]
```diff
@@ -8,16 +8,10 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/gaines-houst
 party:
 - name: Republican
 roles:
```
Comment (maxknutsen) [inline data/ga/retired/Houston-Gaines-e73cb2a6-bf3e-4211-a386-142c18a4f607.yml:10]: @sroomf move to leg.

File: data/ga/retired/J-Collins-bda95ef9-7054-4100-ab95-498abe8b892f.yml [hunk 13]
```diff
@@ -8,16 +8,14 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/collins-j-48
 party:
 - name: Republican
 roles:
-- type: lower
+- end_date: '2023-01-09'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:ga/government
```
Comment (maxknutsen) [inline data/ga/retired/J-Collins-bda95ef9-7054-4100-ab95-498abe8b892f.yml:13]: @sroomf move to leg.

File: data/ga/retired/Josh-Bonner-0e45048c-c1a4-415f-9ee4-5d09ea857e2e.yml [hunk 14]
```diff
@@ -8,16 +8,10 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/bonner-josh-
 party:
 - name: Republican
 roles:
```
Comment (maxknutsen) [inline data/ga/retired/Josh-Bonner-0e45048c-c1a4-415f-9ee4-5d09ea857e2e.yml:10]: @sroomf move to leg.

File: data/ga/retired/Josh-McLaurin-b83b0486-773f-4979-a1ce-e2f7f3e6d7e1.yml [hunk 15]
```diff
@@ -8,16 +8,10 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/mclaurin-jos
 party:
 - name: Democratic
```
Comment (maxknutsen) [inline data/ga/retired/Josh-McLaurin-b83b0486-773f-4979-a1ce-e2f7f3e6d7e1.yml:9]: @sroomf move to leg.

File: data/ga/retired/Kim-Schofield-eff357d0-3187-4a7b-93fe-2d54d010b093.yml [hunk 16]
```diff
@@ -8,13 +8,10 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/schofield-ki
 party:
 - name: Democratic
```
Comment (maxknutsen) [inline data/ga/retired/Kim-Schofield-eff357d0-3187-4a7b-93fe-2d54d010b093.yml:9]: @sroomf move to leg.

File: data/ga/legislature/Marcus-Wiedower-93826349-e5d6-47fc-a7c6-548257629272.yml [hunk 17]
```diff
@@ -0,0 +1,38 @@
...
+given_name: Marcus
+family_name: Wiedower
+email: marcus.wiedower@house.ga.gov
+image: https://www.legis.ga.gov/api/images/default-source/portraits/wiedower-marcus-4956.jpg
+party:
+- name: Republican
+roles:
+- type: lower
+  jurisdiction: ocd-jurisdiction/country:us/state:ga/government
+  district: '121'
```
Comment (maxknutsen) [inline data/ga/legislature/Marcus-Wiedower-93826349-e5d6-47fc-a7c6-548257629272.yml:12]: @sroomf this person was a representative for district 119 from Jan 2019 until Jan 2023.

File: data/ga/legislature/Matt-Hatchett-87ff7d22-1683-4c7f-a2c1-4233b80c2039.yml [hunk 18]
```diff
@@ -0,0 +1,39 @@
+id: ocd-person/87ff7d22-1683-4c7f-a2c1-4233b80c2039
+name: Matt Hatchett
+given_name: Matt
+family_name: Hatchett
+email: matt.hatchett@house.ga.gov
+image: https://www.legis.ga.gov/api/images/default-source/portraits/hatchett-matt-781.jpg
+party:
+- name: Republican
```
Comment (maxknutsen) [inline data/ga/legislature/Matt-Hatchett-87ff7d22-1683-4c7f-a2c1-4233b80c2039.yml:8]: @sroomf this person was been in the house district 150 from January 2011 until 01/09/2023. So they may already have a file elsewhere.

File: data/ga/legislature/Rhonda-Taylor-a9eb583c-60cb-4550-b371-fbad88a6499a.yml [hunk 19]
```diff
@@ -0,0 +1,37 @@
+id: ocd-person/a9eb583c-60cb-4550-b371-fbad88a6499a
+name: Rhonda Taylor
+given_name: Rhonda
+family_name: Taylor
+email: rhonda.taylor@house.ga.gov
+image: https://www.legis.ga.gov/api/images/default-source/portraits/taylor-rhonda-4998.jpg
+party:
+- name: Democratic
+roles:
```
Comment (maxknutsen) [inline data/ga/legislature/Rhonda-Taylor-a9eb583c-60cb-4550-b371-fbad88a6499a.yml:9]: @sroomf this person served in house district 91 from Jan 21 until 01/09/2023.

File: data/ga/legislature/Richard-Smith-d0827b60-7e78-46e8-944a-08e292704805.yml [hunk 20]
```diff
@@ -0,0 +1,38 @@
+id: ocd-person/d0827b60-7e78-46e8-944a-08e292704805
+name: Richard Smith
+given_name: Richard
+family_name: Smith
+email: richard.smith@house.ga.gov
+image: https://www.legis.ga.gov/api/images/default-source/portraits/smith-richard-213.jpg
+party:
+- name: Republican
```
Comment (maxknutsen) [inline data/ga/legislature/Richard-Smith-d0827b60-7e78-46e8-944a-08e292704805.yml:8]: @sroomf this person was in Georgia House district 134 from January 2015 until 01/09/2023 and was in district 131 from January 2005 until January 2015.

File: data/ga/legislature/Rick-Williams-5c79a66d-a3f7-40ce-8114-52224ce6f07a.yml [hunk 21]
```diff
@@ -0,0 +1,35 @@
+id: ocd-person/5c79a66d-a3f7-40ce-8114-52224ce6f07a
+name: Rick Williams
+given_name: Rick
+family_name: Williams
+email: rick.williams@senate.ga.gov
+image: https://www.legis.ga.gov/api/images/default-source/portraits/williams-rick-4905.jpg
+party:
+- name: Republican
+roles:
```
Comment (maxknutsen) [inline data/ga/legislature/Rick-Williams-5c79a66d-a3f7-40ce-8114-52224ce6f07a.yml:9]: @sroomf this person served in house district 145 from January 2017 until 01/09/2023. So they may have another file somewhere.

File: data/ga/retired/Rob-Leverett-29210f3f-ec97-47c8-9492-b81154275109.yml [hunk 22]
```diff
@@ -8,16 +8,10 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/leverett-rob
 party:
 - name: Republican
```
Comment (maxknutsen) [inline data/ga/retired/Rob-Leverett-29210f3f-ec97-47c8-9492-b81154275109.yml:9]: @sroomf move to leg.

File: data/ga/legislature/Sam-Park-bcab0f3a-68fd-4f91-bbc0-bb68b122c044.yml [hunk 23]
```diff
@@ -0,0 +1,38 @@
+id: ocd-person/bcab0f3a-68fd-4f91-bbc0-bb68b122c044
+name: Sam Park
+given_name: Sam
+family_name: Park
+email: sam.park@house.ga.gov
+image: https://www.legis.ga.gov/api/images/default-source/portraits/park-sam-4901.jpg
+party:
+- name: Democratic
+roles:
```
Comment (maxknutsen) [inline data/ga/legislature/Sam-Park-bcab0f3a-68fd-4f91-bbc0-bb68b122c044.yml:9]: @sroomf this person served in the house district 101 from January 2017 until  01/09/2023.

File: data/ga/legislature/Scott-Hilton-91abe321-5ea4-4f17-b8d8-573d7cb913e6.yml [hunk 24]
```diff
@@ -0,0 +1,32 @@
+id: ocd-person/91abe321-5ea4-4f17-b8d8-573d7cb913e6
+name: Scott Hilton
+given_name: Scott
+family_name: Hilton
+email: scott.hilton@house.ga.gov
+image: https://www.legis.ga.gov/api/images/default-source/portraits/hilton-scott-4899.jpg
+party:
+- name: Republican
+roles:
+- type: lower
```
Comment (maxknutsen) [inline data/ga/legislature/Scott-Hilton-91abe321-5ea4-4f17-b8d8-573d7cb913e6.yml:10]: @sroomf this person was in house district 95 between January 2017 and january 2019. They may have another file elsewhere.

File: data/ga/retired/Sharon-Cooper-15db25fc-273b-467f-b8f9-897337f40b65.yml [hunk 25]
```diff
@@ -8,18 +8,10 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/cooper-sharo
 party:
 - name: Republican
 roles:
```
Comment (maxknutsen) [inline data/ga/retired/Sharon-Cooper-15db25fc-273b-467f-b8f9-897337f40b65.yml:10]: @sroomf move to leg.

File: data/ga/retired/Sheila-Jones-b278b399-5ca3-4cd7-bc19-4053166577a2.yml [hunk 26]
```diff
@@ -8,17 +8,10 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/jones-sheila
 party:
 - name: Democratic
 roles:
```
Comment (maxknutsen) [inline data/ga/retired/Sheila-Jones-b278b399-5ca3-4cd7-bc19-4053166577a2.yml:10]: @sroomf move to leg.

File: data/ga/retired/Angela-Moore-b001a55f-61fa-4e62-85fd-ac6c282f2711.yml [hunk 27]
```diff
@@ -8,25 +8,30 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/moore-angela
 party:
 - name: Democratic
 roles:
```
Comment (maxknutsen) [inline data/ga/retired/Angela-Moore-b001a55f-61fa-4e62-85fd-ac6c282f2711.yml:10]: @sroomf move to leg.

File: data/ga/retired/Barry-Fleming-7ea227b9-339a-485a-bfc8-467206ad2457.yml [hunk 28]
```diff
@@ -8,16 +8,21 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/fleming-barr
 party:
 - name: Republican
 roles:
-- type: lower
+- end_date: '2023-01-09'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:ga/government
   district: '121'
```
Comment (maxknutsen) [inline data/ga/retired/Barry-Fleming-7ea227b9-339a-485a-bfc8-467206ad2457.yml:14]: @sroomf move to leg.

File: data/ga/retired/Darshun-Kendrick-1f94316e-fb58-401f-bbfc-5ada0a717b6b.yml [hunk 29]
```diff
@@ -8,19 +8,21 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/kendrick-dar
 party:
 - name: Democratic
```
Comment (maxknutsen) [inline data/ga/retired/Darshun-Kendrick-1f94316e-fb58-401f-bbfc-5ada0a717b6b.yml:9]: @sroomf move to leg.

File: data/ga/retired/Dale-Washburn-ec6cf250-ebcc-4dda-89a5-12da9ebbe51a.yml [hunk 30]
```diff
@@ -8,19 +8,21 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/washburn-dal
 party:
 - name: Republican
 roles:
-- type: lower
+- end_date: '2023-01-09'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:ga/government
```
Comment (maxknutsen) [inline data/ga/retired/Dale-Washburn-ec6cf250-ebcc-4dda-89a5-12da9ebbe51a.yml:13]: @sroomf move to leg.

File: data/ga/retired/David-Knight-1871260f-8daf-43e1-a9ec-09ebcfdbb6a4.yml [hunk 31]
```diff
@@ -8,19 +8,21 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/knight-david
 party:
 - name: Republican
```
Comment (maxknutsen) [inline data/ga/retired/David-Knight-1871260f-8daf-43e1-a9ec-09ebcfdbb6a4.yml:9]: @sroomf move to leg.

File: data/ga/retired/El-Mahdi-Holly-4615e308-db74-408f-89df-02be56b5013b.yml [hunk 32]
```diff
@@ -8,20 +8,21 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/holly-el-mah
 party:
 - name: Democratic
```
Comment (maxknutsen) [inline data/ga/retired/El-Mahdi-Holly-4615e308-db74-408f-89df-02be56b5013b.yml:9]: @sroomf move to leg.

File: data/ga/retired/Gregg-Kennard-f936e87a-99f0-45a7-915b-496bc7574728.yml [hunk 33]
```diff
@@ -8,19 +8,21 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/kennard-greg
 party:
 - name: Democratic
```
Comment (maxknutsen) [inline data/ga/retired/Gregg-Kennard-f936e87a-99f0-45a7-915b-496bc7574728.yml:9]: @sroomf move to leg.

File: data/ga/retired/Patty-Bentley-af51c2db-8871-45a4-803d-314fa2b9d18d.yml [hunk 34]
```diff
@@ -8,19 +8,21 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/bentley-patt
 party:
 - name: Democratic
```
Comment (maxknutsen) [inline data/ga/retired/Patty-Bentley-af51c2db-8871-45a4-803d-314fa2b9d18d.yml:9]: @sroomf move to leg.

File: data/ga/retired/Shelly-Hutchinson-4161e949-6ea2-4df9-8248-cabcf40286ae.yml [hunk 35]
```diff
@@ -8,20 +8,21 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/hutchinson-s
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2023-01-11'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:ga/government
```
Comment (maxknutsen) [inline data/ga/retired/Shelly-Hutchinson-4161e949-6ea2-4df9-8248-cabcf40286ae.yml:13]: @sroomf move to leg.

File: data/ga/retired/Trey-Rhodes-9dfc8ec3-0935-4d1d-877d-e05cf7a5dca5.yml [hunk 36]
```diff
@@ -8,19 +8,21 @@ image: https://www.legis.ga.gov/api/images/default-source/portraits/rhodes-trey-
 party:
 - name: Republican
 roles:
```
Comment (maxknutsen) [inline data/ga/retired/Trey-Rhodes-9dfc8ec3-0935-4d1d-877d-e05cf7a5dca5.yml:10]: @sroomf move to leg.

**Review by maxknutsen — COMMENTED** [review]
@sroomf all good once changes are made, let me know if I can help at all!

File: data/ga/legislature/Colton-Moore-33b353c3-50e9-485f-b154-0d5310da4f2f.yml (hunk 5 above)
Comment (sroomf) [reply data/ga/legislature/Colton-Moore-33b353c3-50e9-485f-b154-0d5310da4f2f.yml:8]: there was one version of his file that was retired and one version in legislature! i think it's a linting error i didn't catch on my first pass, thanks for noticing it!

**Pull request merged**